models/staging/stg_fivetran_log__log.sql

```diff
@@ -1,3 +1,11 @@
+{{ config(
+    partition_by={
```

Comment:
by default this model is a view -- would partitions make a difference here?

Reply:
No, it wouldn't. I actually just did this here to test, because I couldn't actually partition the underlying raw table, which is what we could have people do in reality. This will/should get removed before we merge.